pkg/cmd/bootstrap.go

```diff
@@ -200,7 +208,15 @@ func initiateInteractiveMode(io *BootstrapParameters, client *utility.Client, cm
 		io.GitOpsWebhookSecret = ui.EnterGitWebhookSecret(io.GitOpsRepoURL)
 	}
 	if io.ServiceRepoURL == "" {
-		io.ServiceRepoURL = ui.EnterServiceRepoURL()
+		serviceRepoURL := ui.EnterServiceRepoURL()
+		p, err := url.Parse(serviceRepoURL)
```

Comment:
Since both the URLs have common logic for validation, could we move it to a single function under `validate.go` and have a unit test?

Reply:
ahh thanks, was unaware of that part of the code, Done !!